.evergreen.yml

```diff
@@ -305,6 +327,32 @@ tasks:
         vars:
           test: replica_set_mongod_config
 
+  - name: release_blocker
+    commands:
+      - func: clone
+      - func: release_blocker
+
+  - name: release_operator
+    commands:
+      - func: clone
+      - func: setup_virtualenv
+      - func: release_docker_image
+        vars:
+          old_image: quay.io/mongodb/community-operator-dev
+          new_image: quay.io/mongodb/mongodb-kubernetes-operator
+          image_type: operator
+
+  - name: release_version_upgrade_hook
+    commands:
+      - func: clone
+      - func: setup_virtualenv
+      - func: release_docker_image
+        vars:
+          old_image: quay.io/mongodb/community-operator-version-upgrade-post-start-hook
+          new_image: quay.io/mongodb/mongodb-kubernetes-operator-version-upgrade-post-start-hook
```

Comment:
maybe we can take this opportunity to re-name the dev version, we could just add a `-dev` to the repo name maybe. What do you think?

Reply:
I totally agree! It actually took me a while to figure out the right repo and I kept forgetting it!

Reply:
(decided to keep it like this for now)